packages/components/input-number/src/input-number.ts

```diff
@@ -1,20 +1,16 @@
 import { isNil } from 'lodash-unified'
+import { useSizeProp } from '@element-plus/hooks'
 import { buildProps, isNumber } from '@element-plus/utils'
-import { componentSizes } from '@element-plus/constants'
+import type { ExtractPropTypes } from 'vue'
+import type InputNumber from './input-number.vue'
 
 export const inputNumberProps = buildProps({
-  id: {
-    type: String,
-    default: undefined,
-  },
```

Comment:
**Suggestion:** I think we do not need to update here, since it was left `undefined` intentionally.

Reply:
Done